bin/glm

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env node
+
+const mri = require('mri');
+const axios = require('axios');
+const fs = require('file-system');
+require('dotenv').config();
+
+const owner = process.env.OWNER;
+const repo = process.env.REPOSITORY;
+// eslint-disable-next-line dot-notation
+axios.defaults.headers.common['Authorization'] = `token ${
+  process.env.AUTH_TOKEN
+}`;
+// eslint-disable-next-line dot-notation
+axios.defaults.headers.common['Accept'] =
+  'application/vnd.github.symmetra-preview+json';
+
+const flags = mri(process.argv.slice(2));
+const commands = flags._;
+const command = commands[0];
+
+function printHelp() {
+  console.log(`
+  GITHUB LABEL MANAGER
+
+    Usage: ./bin/glm [command] [option]
+
+      Commands:
+        init        Initializes the label manager. It creates a new label-config.json
+                    file with the current label data of your repository.
+
+        sync        Uploads all changes you've made in the label-config.json file
+                    into the repository to update the label with the given changes.
+  `);
+}
+
+function printMessage(message) {
+  console.log(`
+  GITHUB LABEL MANAGER...
+    ${message}
+  `);
+}
+
+(async () => {
+  try {
+    if (command === 'sync') {
+      const summary = await labelControl();
+      if (typeof summary === 'string') {
+        printMessage(summary);
+      }
+      setTimeout(await writeLabelsConfig, 2000);
+      if (typeof summary === 'object') {
+        printMessage(`Labels synchronized!
+      ${summary.created} created
+      ${summary.deleted} deleted
+      ${summary.updated} updated
+        `);
+      }
+    } else if (command === 'init') {
+      await writeLabelsConfig();
+      printMessage('Labels successfully stored!');
+    } else {
+      printHelp();
+    }
+  } catch (err) {
+    console.error(err);
+    printHelp();
+    process.exit(1);
+  }
+})();
+
+function writeLabelsConfig() {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const labelData = await fetchLabels();
+      const labels = await labelFormatter(labelData);
+      fs.writeFile('./label-config.json', JSON.stringify(labels, null, 2));
+      resolve();
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
+
+function labelControl() {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const labelData = await fetchLabels();
+      const formattedPublishedLabels = await labelFormatter(labelData);
+      fs.readFile('./label-config.json', async (err, data) => {
```

Comment:
Actually we can simply use the synchronous version instead of using callbacks and complicating our lives.

```js
const data = fs.readFileSync('./label-config.json', { encoding: 'utf8' })
```

Reply:
Done 567f587
👍